scripts/generate_lut.py

```diff
@@ -0,0 +1,42 @@
+__doc__ = """
+    To run:
+        cd /path/to/this/file
+        python3 generate_lut.py
+
+    Generates a lookup table in your current working directory.
+    Edit the LUT_NAME, LUT_FUNC, and LUT_SIZE variables    to generate the LUT that you need.
+
+    Note: the size of the LUT in bytes will be 4*LUT_SIZE
+"""
+
+import numpy as np
+
+COL_SIZE = 8
+
+LUT_NAME = "ADC_TO_TEMP_LUT"
+LUT_SIZE = 1 << 12
+def LUT_FUNC(x):
+    return 153 + -0.376*x + 7.36E-04*(x**2) + -8.71E-07*(x**3) + \
+    6.09E-10*(x**4) + -2.55E-13*(x**5) + 6.26E-17*(x**6) + -8.29E-21*(x**7) + 4.55E-25*(x**8)
```

Comment:
Is this the equation that you're going to use to generate the LUT? Like I've said, you can just straight up put in the beta equation here.

To give you an idea of what this `LUT_FUNC` would look like:

```Python
np.exp(...whatever...)*adc_to_voltage_coeff*voltage_to_temp_coeff
```

This will make it so your LUT converts directly from ADC to temperature.